modules/linux_directory_cleaner/manifests/init.pp

```diff
@@ -0,0 +1,92 @@
+class linux_directory_cleaner (
+  Boolean $enabled = true,
+) {
+  # Install the directory_cleaner package using pip3
+  # Doesn't work yet. Pypi issue?
+  exec { 'install_directory_cleaner_linux':
+    command => '/usr/local/bin/pip3 install directory_cleaner==0.2.0',
```

Comment:
@aerickson there is an issue here installing but I think it just has to do with the right whl being uploaded to pypy

[this](https://pypi.pub.build.mozilla.org/pub/directory_cleaner-0.1.0-py3-none-any.whl) doesn't seem to be the one...

```
[rcurran@t-linux64-ms-137 ~]$ /usr/local/bin/pip3 install directory_cleaner==0.2.0
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement directory_cleaner==0.2.0 (from versions: none)
ERROR: No matching distribution found for directory_cleaner==0.2.0

[notice] A new release of pip available: 22.3.1 -> 24.2
[notice] To update, run: python3 -m pip install --upgrade pip
```

Reply:
> @aerickson there is an issue here installing but I think it just has to do with the right whl being uploaded to pypy
> 
> [this](https://pypi.pub.build.mozilla.org/pub/directory_cleaner-0.2.0-py3-none-any.whl) doesn't seem to be the one...
> 
> ```
> [rcurran@t-linux64-ms-137 ~]$ /usr/local/bin/pip3 install directory_cleaner==0.2.0
> Defaulting to user installation because normal site-packages is not writeable
> ERROR: Could not find a version that satisfies the requirement directory_cleaner==0.2.0 (from versions: none)
> ERROR: No matching distribution found for directory_cleaner==0.2.0
> 
> [notice] A new release of pip available: 22.3.1 -> 24.2
> [notice] To update, run: python3 -m pip install --upgrade pip
> ```

I think this is because the root user doesn't have the mozilla pypi server configured. I guess we haven't installed root-level pip3's from our pypi server before.

```bash
root@t-linux64-ms-137:~# su - cltbld
[cltbld@t-linux64-ms-137 ~]$ pip3 install directory_cleaner
Defaulting to user installation because normal site-packages is not writeable
Looking in links: https://pypi.pub.build.mozilla.org/pub/
Collecting directory_cleaner
  Downloading https://pypi.pub.build.mozilla.org/pub/directory_cleaner-0.2.0-py3-none-any.whl (6.6 kB)
Collecting toml<0.11.0,>=0.10.2
  Using cached https://pypi.pub.build.mozilla.org/pub/toml-0.10.2-py2.py3-none-any.whl (16 kB)
Installing collected packages: toml, directory_cleaner
  WARNING: The script directory_cleaner is installed in '/home/cltbld/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed directory_cleaner-0.2.0 toml-0.10.2
[cltbld@t-linux64-ms-137 ~]$ pip3 uninstall directory_cleaner
Found existing installation: directory_cleaner 0.2.0
Uninstalling directory_cleaner-0.2.0:
  Would remove:
    /home/cltbld/.local/bin/directory_cleaner
    /home/cltbld/.local/lib/python3.9/site-packages/directory_cleaner-0.2.0.dist-info/*
    /home/cltbld/.local/lib/python3.9/site-packages/directory_cleaner/*
    /home/cltbld/.local/lib/python3.9/site-packages/pyproject.toml
Proceed (Y/n)? y
  Successfully uninstalled directory_cleaner-0.2.0
[cltbld@t-linux64-ms-137 ~]$ 
```